tests/models/awardsV2/additionalDetails/BaseContractAdditionalDetails-test.js

```diff
@@ -28,4 +28,25 @@ describe('BaseContractAdditionalDetails', () => {
     it('should return -- for null values', () => {
         expect(details.clingerCohenAct).toEqual('--');
     });
+    it('should return the string \'TRUE\' for true boolean values', () => {
+        const trueContractDetails = Object.assign({}, mockIdv.latest_transaction_contract_data, {
+            small_business_competitive: true
+        });
+        const updatedDetailsTrue = Object.create(BaseContractAdditionalDetails);
+        updatedDetailsTrue.populate(trueContractDetails);
+
+        expect(updatedDetailsTrue.smallBusinessCompetitive).toEqual('TRUE');
+    });
+    it('should return the string \'FALSE\' for true boolean values', () => {
+        expect(details.smallBusinessCompetitive).toEqual('FALSE');
+    });
+    it('should return \'--\' for null/undefined values', () => {
+        const nullContractDetails = Object.assign({}, mockIdv.latest_transaction_contract_data, {
+            small_business_competitive: null
+        });
+        const updatedDetailsNull = Object.create(BaseContractAdditionalDetails);
+        updatedDetailsNull.populate(nullContractDetails);
+
+        expect(updatedDetailsNull.smallBusinessCompetitive).toEqual('--');
+    });
```

Comment:
Should we have a test for the other model file as well?

Reply:
Do you mean the awards v1 model?

Reply:
for `src/js/models/v2/awardsV2/BaseParentAwardDetails.js`

Reply:
@ebdabbs 

Merging w/o these added tests as there is basically no logic in there. It would be nice to have tests in place in case someone just `willy nilly` changes the models. We can address this in a follow up PR.